constraint_utils/constraint_basics.py

```diff
@@ -722,6 +746,16 @@ def build_plateau_models_pop(ht: hl.Table, weighted: bool = False) -> Dict[str,
 
 
 def get_all_pop_lengths(ht, prefix: str = 'observed_', pops: List[str] = POPS, skip_assertion: bool = False):
+    """
+    Get the minimum size of the arrays under observed variants column for each population.
+
+    :param ht: High coverage table.
```

Comment:
This doesn't need to be the high coverage Table, it can be any HT that has annotations with a set prefix and list of postfixes of interest where those annotations are arrays

```suggestion
    :param ht: Input Table.
```